Source/Core/DolphinWX/NetPlay/NetPlaySetupFrame.cpp

```diff
@@ -118,14 +118,13 @@ NetPlaySetupFrame::NetPlaySetupFrame(wxWindow* const parent, const CGameListCtrl
 			_("ALERT:\n\n"
 			"Netplay will only work with the following settings:\n"
 			" - DSP Emulator Engine Must be the same on all computers!\n"
-			" - DSP on Dedicated Thread [OFF]\n"
 			" - Manually set the extensions for each Wiimote\n"
 			"\n"
 			"All players should use the same Dolphin version and settings.\n"
 			"All memory cards must be identical between players or disabled.\n"
 			"Wiimote support is probably terrible. Don't use it.\n"
 			"\n"
-			"If connecting directly host must have the chosen UDP port open/forwarded!\n"));
+			"If connecting directly, host must have the chosen UDP port open/forwarded!\n"));
```

Comment:
While you're changing this, maybe add an extra "the" too? ("..., _the_ host must have...")